<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sampling_studio</class>
 <widget class="QMainWindow" name="sampling_studio">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sampling Studio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #007ACC; /* Button background color */
    color :rgb(255, 255, 255);
    padding: 10px;
	border-radius:5px;

    
}

QPushButton:hover {
    background-color: #005CA3; 
}
QPushButton svg {
    fill: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
    <item>
     <widget class="QGroupBox" name="generate_control_group_box">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
         <item>
          <layout class="QVBoxLayout" name="generate_signal_layout" stretch="1,10">
           <item>
            <widget class="QLabel" name="generate_signal_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Generate Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="generate_signal_buttons_and_viewer_layout">
             <item>
              <layout class="QVBoxLayout" name="generate_signal_buttons_layout">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <layout class="QHBoxLayout" name="generated_type_layout" stretch="0,0">
                 <item>
                  <widget class="QLabel" name="generated_type_label">
                   <property name="text">
                    <string>Type</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="generated_type_combo_box">
                   <item>
                    <property name="text">
                     <string>Sine Wave</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Cosine Wave</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="generated_amplitude_layout">
                 <item>
                  <widget class="QLabel" name="generated_amplitude_label">
                   <property name="text">
                    <string>Amplitude</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="generated_amplitude_spin_box">
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="generated_amplitude_layout_2">
                 <item>
                  <widget class="QLabel" name="generated_frequency_label">
                   <property name="text">
                    <string>Frequency</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="generated_frequency_spin_box">
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="generated_signal_name_layout">
                 <item>
                  <widget class="QLabel" name="generated_signal_name_label">
                   <property name="text">
                    <string>Signal Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="generated_signal_name_line_Edit">
                   <property name="text">
                    <string>signal 1</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="add_to_viewers_button">
                   <property name="text">
                    <string>Add To Viewers</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="generated_signal_viewer" native="true"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="viewer_control_layout" stretch="2,2,2">
           <item>
            <widget class="QLabel" name="viewer_controls_label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Viewer Controls</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="viewer_controls_buttons_layout">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="sampling_frequency_layout">
               <item>
                <widget class="QLabel" name="sampling_frequency_label">
                 <property name="text">
                  <string>Sampling Frequency</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="sampling_frequency_slider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="frequency_label">
                 <property name="text">
                  <string>freq</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="freq_combobox">
                 <item>
                  <property name="text">
                   <string>max</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>normal</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="snr_layout">
               <item>
                <widget class="QLabel" name="snr_label">
                 <property name="text">
                  <string>SNR</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="snr_slider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="selected_signal_combo_box"/>
             </item>
             <item>
              <widget class="QPushButton" name="remove_button">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="viewers_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionexit"/>
    <addaction name="upload_signal_button_2"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="upload_signal_buttokaka">
   <property name="text">
    <string>Upload Signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="add_signal_channel2_button">
   <property name="text">
    <string>Channel 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="export_channel1_btn">
   <property name="text">
    <string>Channel 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+1</string>
   </property>
  </action>
  <action name="export_channel2_btn">
   <property name="text">
    <string>Channel 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+2</string>
   </property>
  </action>
  <action name="export_channel12_btn">
   <property name="text">
    <string>Both Channels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+3</string>
   </property>
  </action>
  <action name="upload_signal_button_2">
   <property name="text">
    <string>Add Signal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
